dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1167,6 +1167,9 @@ def test_save_and_load(self):
             data = dp.Data(os.path.join(data_folder, test_file))
             save_profile = UnstructuredProfiler(data)
 
+            # address case where data is read in a way that has no empty lines
+            save_profile._empty_line_count = 1
```

Comment:
Sorry I'm still confused, you say "no empty lines" but its set to 1, can you explain this a bit further? Also why this addition is necessary?

Reply:
TextData currently the entire dataset as a single string. that means `_empty_line_count == 0`. since that is also the default value, this value could be incorrectly saved if we don't test when it is not the default value

Reply:
Can you suggest a different comment?

Reply:
Oh so this way you check later than the _empty_line_count is correctly 0? Maybe say like "change off of default value to ensure it is correctly calculated to be 0"?